parsec/core/sync_monitor.py

```diff
@@ -150,7 +149,7 @@ def set_remote_change(self, entry_id: EntryID) -> bool:
         self.due_time = timestamp()
         return True
 
-    async def tick(self) -> float:
+    async def tick(self, do_sync=True) -> float:
```

Comment:
Why is this `do_sync` argument necessary? What's the point of ticking if not for syncing? At least, let's have a docstring and an explaination

Reply:
During the init step of the monitor, we call tick for the userfs (and also for each workspacefs present). We don't want this tick to actually do sync work (because it means the init will take more time), we only want to know the due_time.

Reply:
Then why use tick to compute the due time? Why not simply make sure that `sync_context.due_time` is always the right value?